frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -84,7 +82,7 @@ const EMBED_THEME_CLASSES = (theme: DashboardUrlHashOptions["theme"]) => {
   }
 };
 
-function EmbedFrame({
+export const EmbedFrame = ({
```

Comment:
https://github.com/metabase/metabase/pull/41973#discussion_r1620209339

Reply:
Ah yeah I see what you mean. I wanted to point at "hey this is a named export" rather than changing to an arrow function. I just wanted to do the arrow function so it's more consistent with some of the newer TS components :smile:

Reply:
Fair enough. I don't think we have ever decided to use either `function functionName` or `const functionName =`, so there are both usages right now.

Reply:
Ah yeah we should probably figure that out. I know `function` has its uses when you need `this` and other things but I think we're at a point in JS-land where we don't need those. So I guess it's up to the FE team